scss/bootstrap/overrides/style/_ui-select.scss

```diff
@@ -137,12 +141,30 @@
     text-transform: none;
   }
 
+  .ui-select-match {
+    display: inline-block;
+    max-width: 90%;
+  }
+  
   .ui-select-search {
     padding-left: 6px !important;
   }
 
   &:after {
-    @include dropdown-arrow();
+    content: '\f002';
+    bottom: 0;
+    font-family: "FontAwesome";
```

Comment:
also, not sure if we should consider putting this in a mixing like the old `dropdown-arrow`